src/main/java/uk/gov/hmcts/sptribs/caseworker/event/CaseworkerIssueCase.java

```diff
@@ -90,22 +90,30 @@ public AboutToStartOrSubmitResponse<CaseData, State> aboutToSubmit(final CaseDet
     }
 
     public SubmittedCallbackResponse submitted(CaseDetails<CaseData, State> details,
-                                            CaseDetails<CaseData, State> beforeDetails) {
+                                               CaseDetails<CaseData, State> beforeDetails) {
+
         final CaseData data = details.getData();
         final CicCase cicCase = data.getCicCase();
-        String caseNumber = data.getHyphenatedCaseRef();
+        final String caseNumber = data.getHyphenatedCaseRef();
 
-        if (!isEmpty(cicCase.getNotifyPartySubject())) {
-            caseIssuedNotification.sendToSubject(details.getData(), caseNumber);
-        }
-        if (!isEmpty(cicCase.getNotifyPartyApplicant())) {
-            caseIssuedNotification.sendToApplicant(details.getData(), caseNumber);
-        }
-        if (!isEmpty(cicCase.getNotifyPartyRepresentative())) {
-            caseIssuedNotification.sendToRepresentative(details.getData(), caseNumber);
-        }
-        if (!isEmpty(cicCase.getNotifyPartyRespondent())) {
-            caseIssuedNotification.sendToRespondent(details.getData(), caseNumber);
+        try {
+            if (!isEmpty(cicCase.getNotifyPartySubject())) {
+                caseIssuedNotification.sendToSubject(details.getData(), caseNumber);
+            }
+            if (!isEmpty(cicCase.getNotifyPartyApplicant())) {
+                caseIssuedNotification.sendToApplicant(details.getData(), caseNumber);
+            }
+            if (!isEmpty(cicCase.getNotifyPartyRepresentative())) {
+                caseIssuedNotification.sendToRepresentative(details.getData(), caseNumber);
+            }
+            if (!isEmpty(cicCase.getNotifyPartyRespondent())) {
+                caseIssuedNotification.sendToRespondent(details.getData(), caseNumber);
+            }
+        } catch (Exception notificationException) {
```

Comment:
Better to catch the exception separately for each notification and prepare the failed parties and include it in the error log and response. In this way, only the failed notification will be reattempted by the user.